README.md

```diff
@@ -18,7 +18,7 @@
   <br/>
   <span>⚡</span>
   <br/>
-  <q><i>It's like Nuxt, but for Chrome Extensions</i></q>
+  <q><i>It's like Nuxt, but for Web Extensions</i></q>
```

Comment:
Haha this was actually for SEO, because people search for "chrome extension" more than "web extension". Not sure how effective it was and this is more accurate lol.

Reply:
Lol, I think people will find the website faster than the github repo tbf